### PR TITLE
added input component

### PR DESCRIPTION
**Pull request by anshikaa12**
inputdoc

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
Looks good to me !

**Comment by AmishMishra11** [comment]
Clean code and proper naming, good job. You can merge it now

**Review by dcs-soni — APPROVED** [review]
Looks good to merge 👍 One suggestion: You can use Prettier to format the CSS code.

File: components/inputs/input.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    border-bottom:2px solid var(--primary-color);
+}
+.no-border-input{
+    border:none;
+}
+/******************/
+.valid-input{
+    background-color: rgb(132, 241, 201,0.5);
+    color:#059862;
+    border: 1px solid #059862;
```
Comment (dcs-soni) [inline components/inputs/input.css:55]: The color code used here can be added in root to avoid repetition and it helps to maintain the code.

**Review by dcs-soni — APPROVED** [review]

File: components/inputs/input.css (hunk 1 above)
Comment (anshikaa12) [reply components/inputs/input.css:55]: yes sure!

**Pull request merged**